internal/oci/container_linux.go

```diff
@@ -104,3 +107,25 @@ func getPidStatDataFromFile(file string) (string, string, error) { //nolint:gocr
 
 	return string(statFields[stateFieldIndex]), string(statFields[startTimeFieldIndex]), nil
 }
+
+// SetRuntimeUser sets the runtime user for the container.
+func (c *Container) SetRuntimeUser(spec *specs.Spec) {
+	runtimeSpec := spec
+	if runtimeSpec.Process == nil {
+		logrus.Infof("Container %s has no process spec", c.ID())
```

Comment:
Do we expect this to always be present? Or most of the time? If so, then perhaps a warning would be better. 

Thoughts?

```suggestion
		logrus.Infof("Container %s is missing process attribute from the runtime specification", c.ID())
```